README.md

```diff
@@ -26,3 +26,30 @@ To set up a single node testnet, [look at the deployment documentation](./docs/d
 
 If you want to deploy a whole cluster, [look at the network scripts](./networks/README.md).
 
+## Dockerized
+
+This is just designed for local testing/CI - DO NOT USE IN PRODUCTION
+
+Build: `docker build  -t wasmd:manual .`
+
+Run:
+
+```sh
+docker volume rm -f wasmd_data
+
+# pick a simple (8 char) passphrase for testing.. you will type it many times
+docker run --rm -it \
+    --mount type=volume,source=wasmd_data,target=/root \
```

Comment:
pp: the `-v` volume syntax is a bit more common. You can use `--volume wasmd_data:/root \` here and below as well.

Reply:
Sure, but all docker docs say it is deprecated.

Change if you wish, but I want to show with docker volumes not familiar dir and this was the only way i knew

Reply:
`volume` is not deprecated but `mount` is recommended. I didn't know that. thanks! Let's leave it like it is now.
> Even though there is no plan to deprecate --volume, usage of --mount is recommended.

https://docs.docker.com/engine/reference/commandline/run/